src/main/webapp/scripts/session-script.js

```diff
@@ -40,9 +70,16 @@ window.onload = function() { main(); }
 function main() {
```

Comment:
This code runs when they are in the session, right?

Reply:
yes